### PR TITLE
Made GetSpriteForFurniture less hard coded

### PR DESCRIPTION
**Pull request by TomMalbran**
This is my solution to make ```GetSpriteForFurniture``` to be less hard coded and is now posible to change the value for any furniture, mentioned in #657.

Eventually we might change this with a sprite animation manager, but for now this can be used for furniture animations.

### CODE REVIEW

**Comment by mikejbrown** [comment]
Much nicer! :+1:

File: Assets/Scripts/Models/Furniture.cs [hunk 1]
```diff
@@ -521,41 +538,41 @@ public void ReadXmlPrototype(XmlReader reader_parent)
...
-                    break;
+                break;
+            
             case "OnInstall":
                 // Called when obj is installed
                 string functionInstallName = reader.GetAttribute("FunctionName");
                 RegisterInstallAction(functionInstallName);
-
                 break;
+
```
Comment (Mizipzor) [inline Assets/Scripts/Models/Furniture.cs]: Could you please clean up trailing whitespace? Not useful and makes for uglier diff/merge. :)

**Comment by bjubes** [comment]
awesome change! sprites should definitely be determined in lua files

**Comment by TomMalbran** [comment]
@Mizipzor Trailing whitespaces removed.

**Comment by Mizipzor** [comment]
Tested. Everything sprite related I can see still works. Good job! :)

**Pull request merged**